massgov/mayflower
# DP-27950 Add nodes to InputTextFuzzy label prop types

## Description
Allow InputTextFuzzy label's to contain both strings and JSX elements. This allows for adding visually hidden instructions for accessibility.

## Related Issue / Ticket
https://github.com/EOANF/budget/issues/557

## Steps to Test


1. Pass a JSX element to `label`
2. Confirm the following warning does not appear in the console  
             
        Warning: Failed prop type: Invalid prop `children` of type `object` supplied to `Label`, expected `string`.
